services/sources/publisher_backlist_service.py

```diff
@@ -23,6 +24,48 @@ def __init__(self):
         
         self.airtable_auth_token = os.environ.get('AIRTABLE_KEY', None)
 
+    def delete_records(
+        self,
+        offset: Optional[int]=None,
+        limit: Optional[int]=None
+    ):
+        filter_by_formula = self.build_filter_by_formula_parameter(deleted=True, full_import=False, start_timestamp=None)
+        
+        array_json_records = self.get_records_array(limit, filter_by_formula)
+
+        for json_dict in array_json_records:
+            if json_dict['records']:
+                for records_value in json_dict['records']:
+                    try:
+                        record_metadata_dict = records_value['fields']
+                        pub_backlist_record = PublisherBacklistMapping(record_metadata_dict)
```

Comment:
do we need to create the publisher back list mapping?

Reply:
I initially used the mapping to get access to the `source_id` directly but then realized that the record_metadata_dict has the Airtable field `DRB Record_ID` which is used as the `source_id` in the DRB record table. Therefore, we don't need to create the mapping.